common/core/github.com/juju/juju/testing/cert-fakejuju.go

```diff
@@ -4,15 +4,15 @@ package testing
 
 import (
 	"io/ioutil"
-	"path/filepath"
 	"os"
+	"path/filepath"
 	"runtime"
 
 	"github.com/juju/loggo"
 
 	gitjujutesting "github.com/juju/testing"
 
-	"github.com/juju/juju/cert"
+	certutils "github.com/juju/utils/cert"
```

Comment:
Is this a 2.1-specific change?  Being in the "common" directory, I'd expect it to work for all supported versions.  Do the other versions compile with this change applied?  The same goes for the 2.1-related patch changes in common/.